Tamil/tastemarshmallows.md

```diff
@@ -0,0 +1,6 @@
+
+Having no idea about where this marshmellow story is going. I happened to choose this folder since it has 
+only one story in it "Yeah". Coming back to the imaginative story I think I would like to 
+
+Taste the marshmellows first [here](https://raw.githubusercontent.com/Sriee/create-your-own-adventure/sriee_story/Tamil/Taste-marshmallows/tastemarshmallows.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.